addon/locales/en/translations.js

```diff
@@ -223,18 +223,24 @@ export default {
                 enterComponentTitle: 'Enter component title',
                 newProjectMessage: 'You have selected to create a new public project for your file. Users will still have access to your file unless the project becomes private.',
                 chooseProject: 'Choose project',
-                projectSelectMessage: 'The list of projects appearing in the selector are projects and components for which you have admin access. Registrations are not included here.',
+                projectSelectMessage: 'The list of projects appearing are projects and components for which you have write access. Registrations are not included here.',
                 noProjectsExistError: 'You have no available projects. Go back to create a new project.',
                 organizeProject: 'Organize',
-                organizeParentNode: 'You can organize your file by storing it in the project or in its own component. If you select "Make a new component", the file will be stored in a new component inside the project. If you select "Use the current component", the file will be stored inside the project. If you are unsure, select "Make a new component".',
-                organizeChildNode: 'You can organize your file by storing it in this component or in its own component. If you select "Make a new component", the file will be stored in a new component inside this component. If you select "Use the current component", the file will be stored inside this component. If you are unusure, select "Make a new component".',
+                organizeParentNode: 'You can organize your file by storing it in the project or its own component.',
+                organizeChildNode: 'You can organize your file by storing it in the component or its own component.',
                 createComponent: 'Create component',
                 couldNotCreateComponent: 'Could not create component. Please try again.',
                 couldNotCreateProject: 'Could not create project. Please try again.',
                 useCurrentComponent: 'Use current component',
                 useCurrentProject: 'Use current project',
-                convertOrCopyMessage: 'You have selected and organized your file. Clicking "Move file" will immediately make changes to your OSF project and move your file. You can continue viewing and editing your file from your project.',
-                noLongerPublicWarning: 'Files moved to private projects/components will no longer be public or discoverable by others.',
+                convertOrCopyMessage: {
+                    project: 'Clicking "Move file" will immediately make changes to your OSF project and move your file.',
+                    component: 'Clicking "Move file" will immediately make changes to your OSF component and move your file.',
```

Comment:
Minor language question: Will moving a file to a project/component actually change that node in any way? This was true for preprints but doesn't seem true for quickfiles.

Reply:
I guess technically it won't change the project/component, but does change its relationships?

Reply:
It will create a log on the node, thus updating its date_modified. So from a user's perspective, it makes a change.